index.html

```diff
@@ -159,141 +156,401 @@ <h2>
+          be no dispatching of events that require the concept of a mouse
+          cursor: namely, `mouseenter`, `mouseleave`, `mouseover`, `mouseout`,
+          `drag`, and `drop`.
+        </p>
+        <p>
+          While in the [=pointer lock state=] if the [=pointer-lock options=]'
+          {{PointerLockOptions/unadjustedMovement}} member is `true` the event
+          coordinates must not be affected by the underlying platform behaviors
+          such as mouse acceleration. In other words, UA must use the APIs
+          provided by the underlying platform to guarantee getting the raw
+          events. If the {{PointerLockOptions}}'
+          {{PointerLockOptions/unadjustedMovement}} member is `false` UA must
+          rely on the default behavior of the underlying platform regarding the
+          mouse acceleration.
+        </p>
+        <p>
+          In the [=pointer lock state=] the system mouse cursor must be hidden.
+          Movement and button presses of the mouse must not cause the window to
+          lose focus.
+        </p>
+        <p class="note">
+          Synthetic mouse events created by application script act the same
+          regardless of lock state.
+        </p>
+      </section>
+      <section data-dfn-for="PointerLockOptions">
+        <h3>
+          `PointerLockOptions` dictionary
+        </h3>
+        <pre class="idl">
+          dictionary PointerLockOptions {
+            boolean unadjustedMovement = false;
+          };
+        </pre>
+        <dl>
+          <dt>
+            <dfn>PointerLockOptions</dfn> dictionary
+          </dt>
+          <dd>
+            <p>
+              The options dictionary to customize how the pointer behaves in
+              the locked mode.
+            </p>
+          </dd>
+          <dt>
+            <dfn>unadjustedMovement</dfn> member
+          </dt>
+          <dd>
+            <p>
+              If this value is set to `true`, then the pointer movements will
+              not be affected by the underlying platform modications such as
+              mouse accelaration.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3>
+          `pointerlockchange` and `pointerlockerror` Events
+        </h3>
+        <p>
+          Two events are used to communicate pointer lock state change or an
+          error in changing state. They are named <dfn>pointerlockchange</dfn>
+          and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
+          for any reason a {{Document/pointerlockchange}} event must be sent.
+        </p>
+        <p>
+          The [=user agents=] must deliver these events by <a data-lt=
+          "queue a task">queuing a task</a> to [=fire an event=] of the
+          appropriate name with its `bubbles` attribute [[DOM]] set to false to
+          the pointer lock [=Event/target=] element's [=Node/node document=].
```

Comment:
this can be cleaned up based on how this is called from the DOM spec.

Reply:
Done